includes/polling/applications/ups-nut.inc.php

```diff
@@ -27,12 +33,11 @@
 
 echo ' '.$name;
 
-list ($charge, $bat_low, $remaining, $bat_volt, $model, $serial, $input_vol, $line_vol, $load) = explode("\n", $ups_nut);
+list ($charge, $remaining, $bat_volt, $model, $serial, $input_volt, $line_volt, $load) = explode("\n", $ups_nut);
 
 $rrd_name = array('app', $name, $app_id);
 $rrd_def = array(
     'DS:charge:GAUGE:600:0:100',
-    'DS:battery_low:GAUGE:600:0:100',
```

Comment:
To try to help out here (though I may not ... ;-)). Yes, I did test it out here, seems to be working well. But to this comment - what is the issue? It is correct that charge and battery_low are two different things. One is a measurement, the other an alarm threshold.

Thanks!